flink-table/flink-sql-client/src/test/java/org/apache/flink/table/client/cli/CliTableauResultViewTest.java

```diff
@@ -156,19 +176,21 @@ public void testBatchResult() {
 
 		view.displayBatchResults();
 		view.close();
-
+		// note: about
```

Comment:
what's this?

Reply:
unused note, I'll clear it.
Thanks for your review,  I’ll address soon.